lib/linguist/heuristics.rb

```diff
@@ -313,6 +313,16 @@ def call(data)
       end
     end
 
+    disambiguate ".props" do |data|
+      if /^(\s*)(<Project|<Import|<Property|<\/Project|<\/Import|<\/Property)/i.match(data) 
```

Comment:
@arfon, xmlns is optional as there are over 3k files: https://github.com/search?p=2&q=extension%3Aprops+xml+version+encoding+OR+project+NOT+xmlns&ref=searchresults&type=Code&utf8=%E2%9C%93. However, I think we should add `<?xml` and `xmlns` checks and remove the closing tags patterns. Does it sound good?